src/database.js

```diff
@@ -80,7 +81,15 @@ export default class Database{
                 }
 
                 done();
-                callback(result);
+                result.rows.forEach(function(element){
```

Comment:
@dsagang, double-check your logic here. Using `forEach` does not overwrite the original array. You might be looking for something like [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map).